docs/filter.rst

```diff
@@ -124,18 +137,16 @@ Here's some useful examples for querying some OSM features:
     |                  |                                                        | | source), but is in reality |
     |                  |                                                        | | not one.                   |
     +------------------+--------------------------------------------------------+------------------------------+
-    | | highways       | | ``type:way and (highway=motorway or``                | | The list of used tags      |
```

Comment:
this old clumsy example can now be written much more nicely with the new `key in (value list)` syntax :sunglasses:

Reply:
true, thanks for updating that 👍